leases/tests/test_lease_queries.py

```diff
@@ -0,0 +1,168 @@
+import pytest
+from graphql_relay import to_global_id
+
+from berth_reservations.tests.utils import (
+    assert_not_enough_permissions,
+    GraphQLTestClient,
+)
+
+client = GraphQLTestClient()
+
+GRAPHQL_URL = "/graphql_v2/"
+
+QUERY_BERTH_LEASES = """
+query GetBerthLeases {
```

Comment:
elsewhere indentation was 2 spaces for these query strings 🤔

Reply:
Agh, damn formatting on multi-line strings.